install.md

```diff
@@ -49,6 +49,10 @@ follow the OS-specific instructions to install pip if you don't know how to do i
 
       sudo apt-get install python-pip
 
+- **Arch / Manjaro (Python3 distributions)**:
```

Comment:
Hmm I think it would be better to add note (maybe inside the ubuntu block - or a new one like you did) for linux users, related to all distros that use python3 by default. The example is good though, and it is good to have it, but I would add a more general comment so we won't add a block for every distro.